File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/MergeDocuments.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class MergeDocuments {
    public List<string> List { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class MergeDocuments {\n");
      sb.Append("  List: ").Append(List).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Pdf-Cloud-SDK-for-Java/src/main/java/com/aspose/pdf/model/MergeDocuments.java
package com.aspose.pdf.model;

import java.util.*;
public class MergeDocuments {
  private List<String> List = new ArrayList<String>();
  /**
	 * getList
	 * Gets List<String>
	 * @return List
	 */
  public List<String> getList() {
    return List;
  }

	/**
	 * setList
	 * Sets List<String>
	 * @param List List<String>
	 */
  public void setList(List<String> List) {
    this.List = List;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class MergeDocuments {\n");
    sb.append("  List: ").append(List).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

